### PR TITLE
Database sim

### PR DESCRIPTION
**Pull request by Weebtrain**
Can now simulate a database with docker-compose using the files in database

### CODE REVIEW

**Review by NMarinoGH — APPROVED** [review]
Good syntax and formatting, working startup files

**Comment by almsam** [comment]
looks good to me, next step is to connect this to python

**Review by MASHAR27 — APPROVED** [review]
Looks well

File: Database/db/init/StructureFile.sql [hunk 1]
```diff
@@ -33,18 +37,27 @@ foreign key (friendId) references User(id),
...
 create table ServerMember (
 userId int,
 serverId int,
 foreign key (userId) references User(id),
-foreign key (serverId) references Server(id),
+foreign key (serverId) references Servers(id),
 primary key (userId,serverId)
 );
+
+create table Message (
```
Comment (NMarinoGH) [inline Database/db/init/StructureFile.sql:56]: add foreign key to userId so we know who sent a message

File: Database/db/init/StructureFile.sql [hunk 2]
```diff
@@ -33,18 +37,27 @@ foreign key (friendId) references User(id),
...
 
 create table ServerMember (
 userId int,
 serverId int,
 foreign key (userId) references User(id),
-foreign key (serverId) references Server(id),
+foreign key (serverId) references Servers(id),
 primary key (userId,serverId)
 );
+
```
Comment (NMarinoGH) [inline Database/db/init/StructureFile.sql:55]: add a ChannelMember table to determine which members can access which channels

**Comment by NMarinoGH** [comment]
Everything looks great. Syntax and formatting are good and I don't see any errors.

**Review by almsam — APPROVED** [review]
Everything seems legit, only issue could be HTML encoding for the date (which should be tested thoroughly)

**Pull request merged**